README.md

```diff
@@ -0,0 +1,8 @@
+# KubeEdge Examples
+
+This repository contains some sample applications and demos which are created to use along with [KubeEdge](https://github.com/kubeedge/kubeedge) platform
+
+The following are the demos that are currently available :-
```

Comment:
Delete this line and create a table below.
|Name | Description | 
Name should have a hyperlink to README of the demo